content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: '.Sqrt()'
+Description: 'Returns the square root of a number.'
+Subjects:
+    - 'Computer Science'
+    - 'Code Foundations'
+Tags:
+    - 'Sqrt'
+CatalogContent:
+    - 'learn-c-sharp'
+    - 'paths/computer-science'
+---
+
+The **`Math.Sqrt()`** method can return the sqaure root of a specified number. If the value of the number happens to be equal to Nan, NegativeInfinity, or PositiveInfinity, that value is returned.
+
+## Syntax
+
+```pseudo
+public static double Sqrt(double x);
+```
+
+The `.Sqrt()` function will can take on one parameter at a time. As shown above _x_ is the value we want to find the sqaure root of.
+
+## Example
+
+The following example will implement how `Math.Sqrt()` will take its given value in the function and return its square root by print it with the `Console.WriteLine()` method.
+
+```cs
+using System;
+
+namespace MySqaureRoot
+    {
+        public class Example
+        {
+            public static void Main(string[] args)
+            {
+                double x = Math.Sqrt(144);
+
+                Console.WriteLine(x);
+            }    
+        }
+    }
+```
+
+This example results in the following output:
```

Comment:
I would suggest having only one example for this method. There isn't really any additional functionality that is being demonstrated by having both examples. I would say the second is more complete, so, you may want to use that one in place of the first and delete the codebyte.

Reply:
Thank you for the suggestions! I went ahead and deleted the codebyte and replaced the first example with the second one. Please let me know if there is anything else I need to add on here!